xsuite/src/world/lsworld.test.ts

```diff
@@ -355,6 +355,35 @@ test.concurrent("LSWorld.setAccount", async () => {
   });
 });
 
+test.concurrent("LSWorld.updateAccount", async () => {
+  using world = await LSWorld.start();
+  const { wallet, wallet2, contract } = await createAccounts(world);
```

Comment:
Let's directly create the account (wallet or contract) without the helper, store the state in a variable and in the final assert, show that the state is the state before with a field changed (similar to what's done in LSWallet.updateAccount).